packages/plugin-ext/src/plugin/plugin-context.ts

```diff
@@ -284,7 +284,7 @@ export function createAPIFactory(
                 return commandRegistry.registerCommand(command, handler, thisArg);
             },
             // eslint-disable-next-line @typescript-eslint/no-explicit-any
-            executeCommand<T>(commandId: string, ...args: any[]): PromiseLike<T | undefined> {
+            executeCommand<T>(commandId: string, ...args: any[]): Thenable<T | undefined> {
```

Comment:
I don't think we should use `Thenable` in this file. I had trouble understanding where it even came from, because it isn't imported from `theia.d.ts`. Turns out it's a side effect of using `vscode-jsonrpc` and somehow the symbol is available here.

Long story short, all those sites are implementations so I think we can/should use `Promise` here.

Reply:
Ah yes i must have overlooked this! Will change! Do you have an opinion on the usage of `PromiseLike`/`Thenable` in the `theia-proposed.d.ts`? Then i will just change this as well, if needed.

Reply:
Reverted the changes for the `plugin-context.ts` file and changed the usages of `PromiseLike` to `Thenable` in `theia-proposed.d.ts`.